src/components/Shared/TabButton.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsxImportSource @emotion/react */
+import { css, type Theme as ThemeType } from '@emotion/react';
+
+type TabButtonProps = {
+  active: boolean;
+  theme: ThemeType;
+  onClick: () => void;
+  icon?: any;
```

Comment:
```suggestion
  icon?: IconType;
```

가능하면 any는 사용하지 않는 스스로의 규칙을 세우고 이번 과제를 진행해보셔도 좋을 거 같습니다1

Reply:
안녕하세요.  
기존에 any 타입으로 처리하던 아이콘 props를 react-icons의 IconType으로 타입을 변경하기 위해
IconType을 import하고 props에 적용했는데, JSX에서 `<Icon />`으로 렌더링할 때 타입스크립트 오류가 발생합니다.  

현재는 임시로 any로 처리하고 있지만, 타입 안전성을 위해 IconType을 제대로 사용하고 싶습니다.  
IconType으로 타입을 지정한 상태에서 JSX 내에서 아이콘 컴포넌트를 문제없이 렌더링하는 올바른 방법이 있을까요?